community/kernel/src/test/java/org/neo4j/kernel/impl/api/KernelStatementTest.java

```diff
@@ -42,31 +36,16 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
-import static org.neo4j.internal.kernel.api.security.SecurityContext.AUTH_DISABLED;
 
 public class KernelStatementTest
 {
-    @Test( expected = TransactionTerminatedException.class )
-    public void shouldThrowTerminateExceptionWhenTransactionTerminated()
```

Comment:
Do we verify this somewhere else?

Reply:
Hmm, I think you're right. We should migrate this test to the new API and the transaction.

Reply:
I added an integration test for this